src/masternodes/rpc_masternodes.cpp

```diff
@@ -25,7 +25,10 @@ UniValue mnToJSON(uint256 const & nodeId, CMasternode const& node, bool verbose)
         obj.pushKV("banTx", node.banTx.GetHex());
         obj.pushKV("state", CMasternode::GetHumanReadableState(node.GetState()));
         obj.pushKV("mintedBlocks", (uint64_t) node.mintedBlocks);
-        obj.pushKV("targetMultiplier", pos::CalcCoinDayWeight(Params().GetConsensus(), node, GetTime()).getdouble());
+        {
+            LOCK(cs_main);
```

Comment:
This LOCK is for `ChainActive().Height()` ? Can just lock to get the height, no need lock for the whole calculation function.
```
int height;
{
    LOCK(cs_main);
    height = ChainActive().Height();
}
.... Use the height here.
```

Reply:
Sure thing, now updated.